deeplinks/index.md

```diff
@@ -66,10 +66,10 @@ Note: alternatively, use username instead of user id, such as [her://user/ZVm2Xp
 Note 2: Alt format that supports only username is [her://profiles/ZVm2XpL7G235](her://profiles/ZVm2XpL7G235)
 
 ## Boosts
-*   [her://boosts](her://boosts) If user has boosts open use boost page, else open purchase page (iOS) / Not supported (Android) 
+*   [her://boosts](her://boosts) If user has boosts open use boost page, else open purchase page (iOS & Android) 
     #### Query Params for her://boosts
 *   [her://boosts?open=buy](her://boosts?open=buy) Open boosts purchase page regardless of number of boosts (iOS) / Not supported (Android) 
```

Comment:
Instead of saying that it's not supported on Android, can we specify that it will open the Boost purchase page? 
The idea is that it can be used in marketing communications regardless of platform, but it will have slightly different behaviors

Reply:
pushed the update